src/services/Api/contacts.ts

```diff
@@ -239,3 +239,113 @@ export async function updateContacts(
     throw body;
   }
 }
+
+export async function fetchDrafts(
+  token: string,
+  orgId: number,
+): Promise<LetterDraft[]> {
+  console.log('made req');
+  const requestOptions: RequestInit = {
```

Comment:
think we should create a util to avoid duplicating this request option code and pass in the method as a parameter

Reply:
This makes sense - under our current method we'd also have to pass in the token - but I know we want to move away from passing the token into these authed methods. We can implement for now with the token passing and then change it later?